packages/xmessage.rb

```diff
@@ -10,7 +10,6 @@ class Xmessage < Package
   depends_on 'xorg_lib'
 
   def self.build
-    system "bash --norc"
```

Comment:
Why?

Reply:
Not a problem with the package, but just curious.

Reply:
I used that line for debugging, don't worry about it.

Reply:
OK. I find that I do that sometimes: I download source tarballs when I'm offline and modify the packages to use them, and then switch it back before I push. It's very useful for packages with large sources.